### PR TITLE
Update release branch

### PR DESCRIPTION
**Pull request by narendasan**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant and/or add your own.

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- This change requires a documentation update

# Checklist:

- [ ] My code follows the style guidelines of this project (You can use the linters)
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas and hacks
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests to verify my fix or my feature
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have added the relevant labels to my PR in so that relevant reviewers are notified

### CODE REVIEW

File: docsrc/tutorials/getting_started_with_fx_path.rst [hunk 1]
```diff
@@ -56,7 +92,7 @@ Explicit batch is the default mode and it must be set for dynamic shape. For mos
 
 For examples of the last path, if we have a 3D tensor t shaped as (batch, sequence, dimension), operations such as torch.transpose(0, 2). If any of these three are not satisfied, we’ll need to specify InputTensorSpec as inputs with dynamic range.
 
-.. code-block:: shell
+c
```
Comment (andi4191) [inline docsrc/tutorials/getting_started_with_fx_path.rst:95]: @narendasan: Is this a typo?

File: noxfile.py [hunk 2]
```diff
@@ -63,31 +66,6 @@ def install_torch_trt(session):
         session.run("python", "setup.py", "develop")
 
 
-def download_datasets(session):
```
Comment (andi4191) [inline noxfile.py:66]: @narendasan @peri044 : Why are we dropping download_datasets in nox?

File: noxfile.py [hunk 3]
```diff
@@ -397,13 +389,3 @@ def l2_trt_compatibility_tests(session):
 def l2_multi_gpu_tests(session):
     """Makes sure that Torch-TensorRT can operate on multi-gpu systems"""
     run_l2_multi_gpu_tests(session)
-
-
-@nox.session(python=SUPPORTED_PYTHON_VERSIONS, reuse_venv=True)
-def download_test_models(session):
```
Comment (andi4191) [inline noxfile.py:403]: How are the models downloaded in nox now?

**Review by andi4191 — APPROVED** [review]
A few questions about nox sessions

File: noxfile.py (hunk 2 above)
Comment (narendasan) [reply noxfile.py:66]: This was changed as part of @peri044's cos similarity overhaul

**Pull request merged**